scripts/west_commands/ncs-provision.py

```diff
@@ -34,6 +36,8 @@ def do_add_parser(self, parser_adder):
             "-k", "--key", type=Path, action='append', dest="keys",
             help="Input .pem file with ED25519 private key"
         )
+        upload_parser.add_argument("-p", "--policy", type=str, help="Keys policy",
+                                   choices=["default", "lock"], default="default")
```

Comment:
It's not clear what `default` means, why don't you use `choices=["revoked", "locked"]` and
```python
  "-r",
  args.policy.uppder(),
  "-v",
```

Reply:
translation table on top

Reply:
agree with @fundakol , don't need extra table, and why to change parameter names from `nrfprovision` tool?
you can use:
```
        upload_parser.add_argument("-p", "--policy", type=str.upper, help="Keys policy",
            choices=["REVOKED", "ROTATING", "LOCKED"], default="REVOKED")
```

Reply:
Name 'revoked' suggests it's dead right away. It should be named differently earlier on but here we are.